Packs/ShiftLeft/Integrations/shiftleft/shiftleft.yml

```diff
@@ -0,0 +1,84 @@
+category: Vulnerability Management
+commonfields:
+  id: ShiftLeft CORE
+  version: -1
+configuration:
+- display: ShiftLeft Organization Id
+  name: org_id
+  type: 0
+  required: true
+  additionalinfo: You can find this under Account Settings -> Org ID
+- display: ShiftLeft Access Token
+  name: access_token
+  type: 4
+  required: true
+  additionalinfo: You can find this under Account Settings -> Access Token
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Integrate ShiftLeft CORE code analysis platform with Cortex XSOAR.
+display: ShiftLeft CORE
+name: ShiftLeft CORE
+script:
+  script: "-"
+  type: python
+  commands:
+  - name: shiftleft-list-apps
+    outputs:
+    - contextPath: ShiftLeft
+      description: API response
+    description: List Apps
+  - name: shiftleft-list-app-findings
+    arguments:
+    - name: app_name
+      required: true
+      default: true
+      description: App name
+    - name: severity
+      auto: PREDEFINED
+      predefined:
+      - critical
+      - moderate
+      - info
+      description: Severity of findings
+      isArray: true
+      defaultValue: critical
+    - name: type
+      auto: PREDEFINED
+      predefined:
+      - vuln
+      - secret
+      - insight
+      - extscan
+      - oss_vuln
+      description: Findings Type
+      isArray: true
+      defaultValue: vuln
+    - name: version
+      description: App version
+    outputs:
+    - contextPath: ShiftLeft
+      description: API response
```

Comment:
Please see my previous comment about the outputs, you should modify them to the correct format.